### PR TITLE
Fix comments

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #22776 .

### Summary of Changes
Fix comments and add a comment for a unclear line

### Testing Instructions
Code review

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on fce37e6a53da34a2afb114d0106c025f0f1158a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31784">issues.joomla.org/tracker/joomla-cms/31784</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on fce37e6a53da34a2afb114d0106c025f0f1158a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31784">issues.joomla.org/tracker/joomla-cms/31784</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31784">issues.joomla.org/tracker/joomla-cms/31784</a>.</sub>

**Comment by richard67** [comment]
@Quy Wrong milestone and label? The PR is for staging, not 4.0-dev.

File: plugins/actionlog/joomla/joomla.php [hunk 1]
```diff
@@ -701,6 +701,7 @@ public function onUserAfterDelete($user, $success, $msg)
 	 */
 	public function onUserAfterSaveGroup($context, $table, $isNew)
 	{
+		// Override context (com_users.group) with the component context (com_users) to pass the chekLoggable
```
Comment (richard67) [inline plugins/actionlog/joomla/joomla.php]:
```suggestion
		// Override context (com_users.group) with the component context (com_users) to pass the checkLoggable
```

**Review by richard67 — CHANGES_REQUESTED** [review]

File: plugins/actionlog/joomla/joomla.php (hunk 1 above)
Comment (richard67) [reply plugins/actionlog/joomla/joomla.php]: Please fix that typo.

**Review by richard67 — APPROVED** [review]

**Comment by chmst** [comment]
Thank you @richard67 and @Quy this is for stable, but the fix is valid also in J4

**Comment by wilsonge** [comment]
As this is just code comments merging this during the RC Window

**Pull request merged**